packages/react-ui-utils/package.json

```diff
@@ -19,7 +19,7 @@
     "babel-eslint": "8.1.2",
     "babel-jest": "22.0.4",
     "babel-plugin-transform-object-rest-spread": "6.26.0",
-    "babel-preset-es2015": "6.24.1",
+    "babel-preset-env": "^1.6.1",
```

Comment:
i noticed all deps except this one are specifically locked in. it might be safer to follow same pattern

Reply:
@lfender6445 - I agree, I'll install the specific version. When the preset is updated with latest language additions we can upgrade it manually as we do with the other `yarn outdated` packages. 👍